pkg/guacanalyze/root.go

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package guacanalyze
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/guacsec/guac/pkg/cli"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	cobra.OnInitialize(cli.InitConfig)
```

Comment:
keep the CLI in https://github.com/guacsec/guac/tree/main/cmd/guacone/cmd

Only move the other logic to this package.